test_scripts/AppIconsFolder/AppIconsFolder_test_set.txt

```diff
@@ -0,0 +1,13 @@
+./test_scripts/AppIconsFolder/ATF_HP_AppIconsFolderMaxSize_compared_with_default.lua Check_positive_case_when_upon_start_SDL_compares_AppIconsFolderMaxSize_in_ini_with_default_value
+./test_scripts/AppIconsFolder/ATF_P_AppIconsFolderMaxSize_less_than_default.lua Check_positive_case_when_upon_start_SDL_compares_less_AppIconsFolderMaxSize_in_ini_than_default_value
+./test_scripts/AppIconsFolder/ATF_P_AppIconsFolderMaxSize_greater_than_default.lua Check_positive_case_when_upon_start_SDL_compares_greater_AppIconsFolderMaxSize_in_ini_with_default_value
+./test_scripts/AppIconsFolder/ATF_HP_AppIconsFolder_is_checked_for_existance.lua Check_positive_case_when_upon_start_SDL_checks_if_AppIconsFolder_exists_and_has_permissions
```

Comment:
@aarotar 
I think this test case can be removed from list, if it do not include any special checks that are not covered with next 4 test cases.